manager-dashboard/app/views/NewProject/index.tsx

```diff
@@ -331,8 +331,7 @@ function NewProject(props: Props) {
                 // NOTE: All the user don't have permission to access draft project
                 // FIXME: The firebase rules need to be changed to perform this on draft project
                 const database = getDatabase();
-                const projectTopicKeyLowercase = (projectTopic?.trim())?.toLowerCase() as string;
-                const projectTopicKey = formatProjectTopic(projectTopicKeyLowercase);
+                const projectTopicKey = formatProjectTopic(name);
```

Comment:
We should still trim and update case to lowercase.
Is that already done in formatProjectTopic function?

Reply:
yes that is already done in `formatProjectTopic` function.